eventActions/tosReminderAction.js

```diff
@@ -0,0 +1,63 @@
+const Discord = require('discord.js')
+table = require('../databaseFiles/tosReminderTable');
+
+const addToDatabase = async function(user,joinTime,reminded=false){
+    table.sync(/*{ force: true }*/).then(async ()=>{
+        await table.create({
+            user_id: user.id,
+            joinTime: joinTime,
+            reminded: reminded
+        }).catch(err =>{
+            console.error('Tosreminder error: ' + err )
+        })
+    })
+}
+const removeFromDatabase = async function(user_id){
+    await table.destory({
```

Comment:
`destory` is misspelled - it should be `destroy`. Please test your changes to catch typos like this before review.